### PR TITLE
[ISSUE 376] Rename command line options for stimulusfile (-s), deserializefile(-r) and serializefile(-w)

### PR DESCRIPTION
**Pull request by Andrzej-Dawiec**
- Renames command line options for stimulusfile (-s), deserializefile(-r) and serializefile(-w)
- Updates the command line option in the documentation under Student Quickstart
- Adds command line option in the documentation under Quickstart

### CODE REVIEW

File: Simulator/Core/Core.cpp [hunk 1]
```diff
@@ -55,15 +55,15 @@ bool Core::parseCommandLine(string executableName, string cmdLineArguments)
    if ((cl.addParam("configfile", 'c', ParamContainer::filename, "parameter configuration filepath")
         != ParamContainer::errOk)
 #if defined(USE_GPU)
-       || (cl.addParam("device", 'd', ParamContainer::regular, "CUDA device id")
+       || (cl.addParam("device", 'g', ParamContainer::regular, "CUDA device id")
```
Comment (stiber) [inline Simulator/Core/Core.cpp:58]: To make the hint explain more clearly why it's `-g`, let's make it `"CUDA GPU device ID"`

File: Simulator/Core/Core.cpp [hunk 2]
```diff
@@ -55,15 +55,15 @@ bool Core::parseCommandLine(string executableName, string cmdLineArguments)
...
            != ParamContainer::errOk)
-#endif   // USE_GPU
-       || (cl.addParam("stimulusfile", 's', ParamContainer::filename, "stimulus input filepath")
+#endif   // USE_GPU changed 'd' to 'g' option
```
Comment (stiber) [inline Simulator/Core/Core.cpp:60]: I don't think it's necessary to have a note about the change in the comment here.

File: docs/Developer/StudentSetup.md [hunk 3]
```diff
@@ -86,9 +86,9 @@ Other external guides for remote VS Code development:
 The following options are available:
 
 - `-c, --configfile=<configfile>`: Specifies the filepath for the parameter configuration file. This file is required.
-- `-s, --stimulusfile=<stimulusfile>`: Specifies the filepath for the stimulus input file. This file is required.
-- `-r, --deserializefile=<deserializefile>`: Specifies the filepath for simulation deserialization. Enabling this option allows you to deserialize a previous simulation.
-- `-w, --serializefile=<serializefile>`: Specifies the filepath for simulation serialization. Enabling this option allows you to serialize the simulation.
+- `-i, --stimulusfile=<stimulusfile>`: Specifies the filepath for the stimulus input file. This file is required.
+- `-d, --deserializefile=<deserializefile>`: Specifies the filepath for simulation deserialization. Enabling this option allows you to deserialize a previous simulation.
+- `-s, --serializefile=<serializefile>`: Specifies the filepath for simulation serialization. Enabling this option allows you to serialize the simulation.
```
Comment (stiber) [inline docs/Developer/StudentSetup.md:91]: Is there a note about the `-g` option in this file?

File: docs/User/quickstart.md [hunk 4]
```diff
@@ -50,7 +50,22 @@ As a quick start and sanity test, let's run a small, prepackaged simulation to m
...
+    $ ./cgraphitti -help
+    ```
+
+The following options are available:
+
+- `-c, --configfile=<configfile>`: Specifies the filepath for the parameter configuration file. This file is required.
+- `-i, --stimulusfile=<stimulusfile>`: Specifies the filepath for the stimulus input file. This file is required.
+- `-d, --deserializefile=<deserializefile>`: Specifies the filepath for simulation deserialization. Enabling this option allows you to deserialize a previous simulation.
+- `-s, --serializefile=<serializefile>`: Specifies the filepath for simulation serialization. Enabling this option allows you to serialize the simulation.
+- `-v, --version`: Outputs the current git commit ID and exits.
```
Comment (stiber) [inline docs/User/quickstart.md:65]: Is there a note about the `-g` option in this file?

**Review by stiber — CHANGES_REQUESTED** [review]

**Comment by jardiamj** [comment]
I took care of the requested changes to attempt merging this PR.

**Review by stiber — APPROVED** [review]

**Pull request merged**